app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.tsx

```diff
@@ -70,26 +71,23 @@ const TableAudience = ({ experiment }: TableAudienceProps) => {
             </td>
           </tr>
         )}
-        {experiment.jexlTargetingExpression !== "" && (
-          <tr>
-            <th>Full targeting expression</th>
-            <td
-              data-testid="experiment-target-expression"
-              className="text-monospace"
-            >
-              <code>
-                <pre>{experiment.jexlTargetingExpression}</pre>
-              </code>
-            </td>
-          </tr>
-        )}
+        {experiment.jexlTargetingExpression &&
```

Comment:
Interesting&hellip; Was this an issue before, or is this just so we don't try and JSON.stringify `null` or `undefined`? Curious why the extra check.

Reply:
That's how I am reading it, you'll get into trouble if you try to stringify a null value

Reply:
TypeScript wanted `experiment.jexlTargetingExpression!` sent into `<FormattedJson json={` to assert it's not undefined so I added the check here instead since we shouldn't display it otherwise.